tests/toranj/openthread-core-toranj-config.h

```diff
@@ -392,6 +392,22 @@
  */
 #define OPENTHREAD_CONFIG_IP6_SLAAC_ENABLE    1
 
+/**
+ * @def RADIO_CONFIG_SRC_MATCH_SHORT_ENTRY_NUM
+ *
+ * The number of short source address table entries.
+ *
+ */
+#define RADIO_CONFIG_SRC_MATCH_SHORT_ENTRY_NUM OPENTHREAD_CONFIG_MLE_MAX_CHILDREN
```

Comment:
`RADIO_CONFIG_SRC_MATCH_SHORT_ENTRY_NUM` is not core config, maybe we should move these definitions into *examples/platforms/utils/soft_source_match_table.c*.

Reply:
Currently, all `example/platforms/utils` related configuration macros are defined in  `openthread-core-xxx-config.h`. So I defined the macro `RADIO_CONFIG_SRC_MATCH_SHORT_ENTRY_NUM` in  the core config file. I submitted an issue #4223 to discuss this topic. If we should move `example/platforms/utils` related configuration macros to other files. I will submit another PR to resolve this issue.

Reply:
To address this specific issue, is it better to set the default value to `OPENTHREAD_CONFIG_MLE_MAX_CHILDREN` in: 
https://github.com/openthread/openthread/blob/cc91b2c72b6b1037302e662d90e7ed0ff674cab2/examples/platforms/utils/soft_source_match_table.h#L49
and

https://github.com/openthread/openthread/blob/cc91b2c72b6b1037302e662d90e7ed0ff674cab2/examples/platforms/utils/soft_source_match_table.h#L53

Reply:
Updated.